docs/howto/openstack/octavia/lbaas-tcp.md

```diff
@@ -0,0 +1,742 @@
+    assignment, click the green _Create and Assign_ button.
+
+    ![Configure attachment](assets/shot-21.png)
+
+    As you may see, your load balancer now has a floating IP. You can now
+    use that to test the balancer and make sure it works as expected.
+
+    ![Floating IP attached to load balancer](assets/shot-22.png)
+=== "OpenStack CLI"
+    First, create a new floating IP:
+
+    ```bash
+    openstack floating ip create ext-net
+    ```
+
+    ```text
+    +---------------------+--------------------------------------+
+    | Field               | Value                                |
+    +---------------------+--------------------------------------+
+    | created_at          | 2023-01-16T20:25:25Z                 |
+    | description         |                                      |
+    | dns_domain          | None                                 |
+    | dns_name            | None                                 |
+    | fixed_ip_address    | None                                 |
+    | floating_ip_address | 198.51.100.129                       |
+    | floating_network_id | 2aec7a99-3783-4e2a-bd2b-bbe4fef97d1c |
+    | id                  | 69457661-197e-444a-b9ce-54394263dafa |
+    | name                | 198.51.100.129                       |
+    | port_details        | None                                 |
+    | port_id             | None                                 |
+    | project_id          | dfc700467396428bacba4376e72cc3e9     |
+    | qos_policy_id       | None                                 |
+    | revision_number     | 0                                    |
+    | router_id           | None                                 |
+    | status              | DOWN                                 |
+    | subnet_id           | None                                 |
+    | tags                | []                                   |
+    | updated_at          | 2023-01-16T20:25:25Z                 |
+    +---------------------+--------------------------------------+
+    ```
+
+    Then, load the ID of the new floating IP to a variable:
+
+    ```bash
+    FLOAT_IP_ID=$(openstack floating ip list \
+        --floating-ip-address 198.51.100.129 -c ID -f value)
+    ```
+
+    Do the same for the VIP port ID of your load balancer:
+
+    ```bash
+    VIP_PORT_ID=$(openstack loadbalancer show mylb \
+        -c vip_port_id -f value)
+    ```
+
+    Finally, assign the floating IP to your load balancer like so:
+
+    ```bash
+    openstack floating ip set --port $VIP_PORT_ID $FLOAT_IP_ID
+    ```
+
+## Testing the load balancer
+
+Each of the two test servers runs `ncat` to continuously listen
+on port 61234/TCP and respond to clients with a simple message,
+revealing its hostname:
+
+```bash
+ncat -kv -l 61234 -c 'echo Yello from $(hostname)!'
+```
+
+Since the load balancer has a floating IP (`198.51.100.129`,
+in our example). and we know the port it listens to, we can
```

Comment:
Looks like a full-stop slipped in here instead of a comma; sorry for not catching this earlier.

```suggestion
in our example), and we know the port it listens to, we can
```

Reply:
**I'm** sorry for not seeing that :/